### ISSUE TITLE
Can't run localGPT on linux

### ISSUE THREAD

**Issue by ffffff4** [PromtEngineer/localGPT#643]
Hi,
I have tried with few 7B models, but I had no success with none of them.

The errors I get are:
  pydantic.error_wrappers.ValidationError: 1 validation error for LLMChainllm
none is not an allowed value (type=type_error.none.not_allowed)
Full log: [localgpt.log](https://github.com/PromtEngineer/localGPT/files/13335036/localgpt.log)
[constants.py](https://github.com/PromtEngineer/localGPT/files/13335048/constants.py.txt)

The models I tried were:
models--NousResearch--Llama-2-7b-chat-hf
models--TheBloke--Llama-2-7b-Chat-GGUF
models--TheBloke--Mistral-7B-Instruct-v0.1-GGUF

I did not change the source documents.
My computer has a NV GPU but I tried to run localGPT using CPU.
Maybe I missed something during installation?
Thank you.

**Comment by MadMartian** [comment]
I'm just another user, but I was able to get it running _mostly_ well on either GPU or CPU on Linux at 15e96488b67eb4145d743f5ef4f3cc9e7102bbb7 using `Pipenv`

**Comment by jeremycheminf** [comment]
I have the same problem , trying different version of way to install llama-cpp-python (most recent or 0.1.83) but still getting the same error asavobe.

**Comment by MadMartian** [comment]
Yes, I think in-fact it was `llama-cpp-python` I had to install, I have version 0.2.11 installed in my pipenv.
